src/libs/antares/benchmarking/file_content.h

```diff
@@ -20,6 +21,7 @@ class FileContent
     void addDurationItem(const std::string& name, unsigned int duration, int nbCalls);
 
 private:
+    std::mutex pMutex;
```

Comment:
may be renaming pMutex to pSectionsMutex would be clearer.

Reply:
sure